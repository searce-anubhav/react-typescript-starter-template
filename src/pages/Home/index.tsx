import * as React from 'react';
import Button from 'components/Button';
import { Link } from 'react-router-dom';

const Home: React.FC = (): JSX.Element => {
	return (
		<React.Fragment>
			<Link to='/login'>
				<Button colorScheme='teal'>Go to Login</Button>
			</Link>
		</React.Fragment>
	);
};

export default Home;
