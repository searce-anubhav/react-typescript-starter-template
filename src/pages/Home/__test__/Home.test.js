import React from 'react';
import renderer from 'react-test-renderer';
import Home from 'pages/Home';
import { MemoryRouter } from 'react-router-dom';

test('Render Home Snapshot and Match', () => {
	const component = renderer.create(
		<MemoryRouter>
			<Home></Home>
		</MemoryRouter>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
