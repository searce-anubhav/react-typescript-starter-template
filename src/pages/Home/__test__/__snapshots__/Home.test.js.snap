// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home Snapshot and Match 1`] = `
<a
  href="/login"
  onClick={[Function]}
>
  <button
    className="chakra-button css-5c2176"
    type="button"
  >
    Go to Login
  </button>
</a>
`;
